python/cuml/_thirdparty/sklearn/preprocessing/_imputation.py

```diff
@@ -91,6 +91,72 @@ def _most_frequent(array, extra_value, n_repeat):
     return value
 
 
+def _masked_median(arr, masked_value):
```

Comment:
I'd pull this out of _imputation.py, since the core idea was that these files have come directly from sklearn with tiny edits. Could be in "thirdparty_adapters.py" which is where we have other similar functions

Reply:
Also, I would rename it to `_masked_column_medium` since it is very different from just like np.median in signature

Reply:
of course when you move it don't forget to update np->cp!

Reply:
Are we okay with moving `_get_mask` as well to avoid the circular import? It looks like in 0.23.1, it's in [utils/_mask.py](https://github.com/scikit-learn/scikit-learn/blob/0.23.1/sklearn/utils/_mask.py) rather than `_imputation.py` anyway.